spec/rubocop/config_loader_spec.rb

```diff
@@ -291,6 +291,27 @@
       configuration = load_file
       expect(configuration).to eq({})
     end
+
+    context 'when SafeYAML is required' do
+      before do
+        create_file(configuration_path, [
+          'Style/WordArray:',
+          "  WordRegex: !ruby/regexp '/\\A[\\p{Word}]+\\z/'"
+        ])
+      end
+
+      it 'de-serializes Regexp class' do
+        # Load SafeYAML in different memory space
+        pid = ::Process.fork {
+          require 'safe_yaml'
```

Comment:
The only problem I see is that after this point `safe_yaml` will be loaded for the rest of the specs.